app/web/features/communities/newEvents/EventsList.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from "@material-ui/core";
+import { EventsType } from "features/queryKeys";
+import { useTranslation } from "i18n";
+import { COMMUNITIES, GLOBAL } from "i18n/namespaces";
+import mockEvents from "test/fixtures/events.json";
+
+import EventItem from "./EventItem";
+
+interface EventListProps {
+  eventType: EventsType;
+}
+
+const EventsList = ({ eventType }: EventListProps) => {
+  const { t } = useTranslation([GLOBAL, COMMUNITIES]);
+
+  const renderEventItems = () => {
+    return mockEvents.map((event) => (
```

Comment:
mockEvents in the list component?

Reply:
Yep as I described in the description of the PR this PR is just the design, so using mock events. Logic comes next.